### DISCUSSION TITLE
Has anyone been succesful in installing google assistant on the pc, windows 10?

### DISCUSSION THREAD

**Discussion by Evolutionist1969** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#853] [General]
Can anyone help me get google assistant on my pc. I have tried several tutorials but no success.

**Comment by sumeshir26** [comment]
https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Setup-Authentication-for-Google-Assistant-Unofficial-Desktop-Client
